test/integration/api/offline-entities.js

```diff
@@ -1126,35 +1126,35 @@ describe('Offline Entities', () => {
           body.currentVersion.data.should.eql({ status: 'checked in' });
           body.currentVersion.label.should.eql('auto generated');
           body.currentVersion.branchId.should.equal(branchId);
+          body.currentVersion.branchBaseVersion.should.equal(1);
           should.not.exist(body.currentVersion.baseVersion);
-          should.not.exist(body.currentVersion.branchBaseVersion); // No base version because this is a create, though maybe this should be here.
           should.not.exist(body.currentVersion.trunkVersion);
         });
 
       backlogCount = await container.oneFirst(sql`select count(*) from entity_submission_backlog`);
       backlogCount.should.equal(0);
 
+      await asAlice.get(`/v1/projects/1/datasets/people/entities/12345678-1234-4123-8234-123456789ddd`)
+        .expect(200)
+        .then(({ body }) => {
+          body.currentVersion.data.should.eql({ status: 'checked in' });
+          should(body.conflict).equal(null); // conflict should be null after update-as-create
+        });
+
       // First submission creates the entity, but this will be processed as an update
       await asAlice.post('/v1/projects/1/forms/offlineEntity/submissions')
-        .send(testData.instances.offlineEntity.two
-          .replace('branchId=""', `branchId="${branchId}"`)
-        )
+        .send(testData.instances.offlineEntity.two)
         .set('Content-Type', 'application/xml')
         .expect(200);
 
       await exhaust(container);
 
-      // In the default behavior, attempting create on an entity that already exists causes a conflict error.
-      await asAlice.get('/v1/projects/1/forms/offlineEntity/submissions/two/audits')
-        .expect(200)
-        .then(({ body }) => {
-          body[0].details.errorMessage.should.eql('A resource already exists with uuid value(s) of 12345678-1234-4123-8234-123456789ddd.');
-        });
-
       await asAlice.get(`/v1/projects/1/datasets/people/entities/12345678-1234-4123-8234-123456789ddd`)
         .expect(200)
         .then(({ body }) => {
-          body.currentVersion.version.should.equal(1);
+          body.currentVersion.version.should.equal(2);
+          body.currentVersion.data.should.eql({ age: '20', status: 'new', first_name: 'Megan' });
+          body.conflict.should.equal('soft'); // this should be marked as a soft conflict
```

Comment:
I think it'd be useful to make an assertion about `baseVersion` as well, since normally the `baseVersion` associated with an entity-create submission is `null`.